### DISCUSSION TITLE
Share one photo via link

### DISCUSSION THREAD

**Discussion by litecactus** [ente-io/ente#4132] [Enhancements]
So when I want to send a friend one (or a couple) of photos I need to create an album and share that....I just want to be able to select a photo, or a couple of photos and send them a link (perhaps with an optional password and/or an expiry date). If I send one photo, I want it to just open in the browser or if I share a couple of photos I want them to share as a .zip.

**Comment by taivlam** [comment]
This should be possible now with the "Send link" option on the mobile Ente Photos apps.
